src/test/java/com/microsoft/sqlserver/jdbc/preparedStatement/BatchExecutionWithBulkCopyTest.java

```diff
@@ -1489,6 +1489,47 @@ public void testBulkInsertStringWhenNotSentAsUnicode() throws Exception {
         }
     }
 
+    /**
```

Comment:
This is very duplicative, as were the previous two tests added in #2704.

This new code can be removed and the previous two tests improved. They can exercise accented characters in their test data since those are supported in LATIN collations. Additionally, think about how the test can demonstrate/validate any expected behavior when inserting Unicode characters into the various column types with the varying connection options (be sure to use Unicode characters in the test data). Also, define the test strings as string constants to avoid string duplication in the code.